### PR TITLE
Finish docker

### PR DESCRIPTION
**Pull request by naphets123**
This should give a full dockerized starting point for development, setting up the database

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,11 @@
 FROM ubuntu:18.04
```
Comment (HaGuesto) [inline Dockerfile:1]: could you maybe use the python docker images? The ubuntu image takes up a lot of space.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+        build: .
         ports:
           - "5000:5000"
         network_mode: "host"
+        volumes:
+          - .:/code
+        environment:
+          FLASK_ENV: development
     mysql:
         image: mysql/mysql-server:latest
```
Comment (HaGuesto) [inline docker-compose.yml:13]: Please use the mysql version in production (5.7)

Comment (HaGuesto) [inline docker-compose.yml:13]: add build commands to drop existing databases and then load init.sql

File: Dockerfile (hunk 1 above)
Comment (naphets123) [reply Dockerfile:1]: I just experimented with some alternative images, there are strong advices not to user alpine, there are issues with some packages(libmysqlclient-dev) for which i couldn't find a quick fix at the moment.

Comment (HaGuesto) [reply Dockerfile:1]: I just thought maybe use the python docker image
https://hub.docker.com/_/python?tab=description

**Pull request merged**